src/type/__tests__/enumType.js

```diff
@@ -59,7 +66,31 @@ describe('Type System: Enum Values', () => {
     }
   });
 
-  var schema = new GraphQLSchema({ query: QueryType });
+  var MutationType = new GraphQLObjectType({
+    name: 'Mutation',
+    fields: {
+      favoriteEnum: {
+        type: FavoriteType,
+        args: { color: { type: ColorType } },
+        resolve(value, color) { return color; }
```

Comment:
the second arg here is a little misleading to call `color`. It's the object of all provided arguments, you should be explicit and use `resolve(value, { color })` to destructure the color key from the arguments object.